tinkergraph-gremlin/src/test/java/org/apache/tinkerpop/gremlin/tinkergraph/TinkerWorld.java

```diff
@@ -109,6 +109,11 @@ else if (graphData == LoadGraphWith.GraphData.GRATEFUL)
         public AbstractTinkerGraph open(final Configuration configuration) {
             return TinkerGraph.open(configuration);
         }
+
+        @Override
+        public boolean useParametersLiterally() {
+            return false;
```

Comment:
doesn't there need to be another test configuration that validates with `true`? am i missing that somewhere?

Reply:
I will add separate worlds and feature test classes for enabling and disabling parameters similar to the remote feature tests.